jax/_src/checkify.py

```diff
@@ -785,12 +785,19 @@ def assert_discharge_rule(error, enabled_errors, pred, code, payload, *, msgs):
 
 ErrorCategory = enum.Enum('ErrorCategory', ['NAN', 'OOB', 'DIV', 'USER_CHECK'])
 
-user_checks = frozenset({ErrorCategory.USER_CHECK})
+
+user_checks = frozenset({ErrorCategory.USER_CHECK})  #: Enables all :func:`check` assertions.
+#: Checks if all floating-point operations output non-NaN values.
```

Comment:
I'm actually unsure of how to add documentation to the ErrorCategory sets in a way that the autodocs generation understands. I thought I could do it with these doc-comments (`#: ...`) but they're not rendering. I'll have another look at documenting "module level variables" but maybe I'm missing something obvious?